<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Simulation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始仿真</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Algorithm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>朴素的轮盘赌算法</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄蜂群算法</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DFS算法</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>算法</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>机器人数量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Robot_num">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Map_coverage">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>360</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地图覆盖率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平均重复率</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Repetition_rate">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>360</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="Map_ylen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Map_xlen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>横向格子数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>纵向格子数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Make_map">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始制作地图</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Map_filename">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地图文件名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
